cmd/kueue/main.go

```diff
@@ -344,7 +344,11 @@ func setupControllers(ctx context.Context, mgr ctrl.Manager, cCache *cache.Cache
 		}
 	}
 
-	if failedWebhook, err := webhooks.Setup(mgr); err != nil {
+	namespace := configapi.DefaultNamespace
```

Comment:
I think the namespace is defaulted if empty.

https://github.com/kubernetes-sigs/kueue/blob/main/apis/config/v1beta1/defaults.go#L71

I think you could just pass in the *cfg.Namespace here.